benchmark/benchmark.go

```diff
@@ -27,6 +27,9 @@ import (
 	"io"
 	"log"
 	"net"
+	"time"
+
+	"math/rand"
```

Comment:
Please group these imports together. You also don't rename this import, which is what I think we should do for main.go as well.

Reply:
done